docs/networks/arbitrum-sepolia.md

```diff
@@ -0,0 +1,113 @@
+# Testnet Configuration
+
+The Graph Network's testnet is on Arbitrum Sepolia (eip155:421614). Sepolia network information can be found at https://testnet.thegraph.com/explorer?chain=arbitrum-sepolia.
+
+## Latest Releases
+
+| Component       | Release                                                                         |
+| --------------- | ------------------------------------------------------------------------------- |
+| contracts       | [5.3.3](https://github.com/graphprotocol/contracts/releases/tag/v5.3.3)         |
+| indexer-agent   | [0.20.22](https://github.com/graphprotocol/indexer/releases/tag/v0.20.22)       |
+| indexer-cli     | [0.20.22](https://github.com/graphprotocol/indexer/releases/tag/v0.20.22)       |
+| indexer-service | [0.20.22](https://github.com/graphprotocol/indexer/releases/tag/v0.20.22)       |
+| graph-node      | [v0.33.0](https://github.com/graphprotocol/graph-node/releases/tag/v0.33.0)     |
+
+## Network Parameters
+
+| Parameter                   | Value             |
+| --------------------------- | ----------------- |
+| Epoch length                | ~ 2h (554 blocks) |
+| Maximum allocation lifetime | ~ 9h (4 epochs)   |
+
+## Contracts & accounts
+
+| Name               | Address                                                                                                                        |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------ |
+| Graph Token (GRT ) | [`0xf8c05dCF59E8B28BFD5eed176C562bEbcfc7Ac04`](https://sepolia.etherscan.io/address/0xf8c05dCF59E8B28BFD5eed176C562bEbcfc7Ac04) |
+| Staking            | [`0x865365C425f3A593Ffe698D9c4E6707D14d51e08`](https://sepolia.etherscan.io/address/0x865365C425f3A593Ffe698D9c4E6707D14d51e08) |
+
+
+[comment]: <> (TODO: CHECK THESE)
+| Data Edge          | [`0xEFC8D47673777b899f2FB597C6FC0E87ecce98Cb`](https://sepolia.etherscan.io/address/0xEFC8D47673777b899f2FB597C6FC0E87ecce98Cb) |
```

Comment:
This address is for the L1 `EventfulDataEdge`. The Arbitrum Sepolia address is `0x9b9402939133F27c6eba81a321dfBFa1feE6714E`